src/js/outreach/apps/verify_app.js

```diff
@@ -15,19 +15,26 @@ import {
 } from 'js/outreach/views/dialog_views';
 
 export default App.extend({
-  onStart({ actionId }) {
-    this.actionId = actionId;
+  beforeStart({ actionId }) {
+    return getPatientInfo({ actionId });
+  },
+  onStart(options, patient) {
+    this.patientPhoneEnd = patient.phone_end;
+    this.patientId = patient.relationships.patient.data.id;
```

Comment:
I'm assuming there's a better way to handle this data... so need to look into that.

Reply:
should probably use `get`s.  `phone_end` will also be on `attributes` as this'll be a fully jsonapi.  I'm not sure if this is a now thing, but we're coming up to being close to needing to setup an outreach entities using the backbone models.

Reply:
I just updated to use `get`.

I'm down to come back after this is deployed and switch over to use backbone models.